biz.aQute.repository/test/aQute/bnd/deployer/repository/FindProvidersTest.java

```diff
@@ -66,7 +66,6 @@ public static void testReadGZippedStream() throws Exception {
 		Capability identityCap = caps.iterator().next();
 
 		List<Capability> contentCaps = identityCap.getResource().getCapabilities("osgi.content");
-		assertNotNull(contentCaps);
```

Comment:
There is nothing wrong with asserting non null. Why remove? Same for the other nonnull checks you removed.